content/systems/filecoin_markets/retrieval_market/_index.md

```diff
@@ -9,89 +9,75 @@ weight: 2
-- When the provider requires payment to proceed, it sends payment request and does not send any more blocks
-- The client creates and stores a payment voucher off-chain
-- The client responds to the provider with a reference to the payment voucher
-- The provider redeems the payment voucher off-chain
-- The provider resumes sending blocks
-- The client consumes blocks until payment is required again
-- The process continues until the end of the query
-
-## Deal Flow (V1)
-
-{{<svg src="retrieval_flow_v1.mmd.svg" title="Retrieval Flow - V1" >}}
-
-The evolved protocol for proposing and accepting a deal will work as follows:
+The evolved Filecoin Retrieval Market protocol, currently in use, for proposing and accepting a deal works as follows:
 
 - The client finds a provider of a given piece with `FindProviders()`.
 - The client queries a provider to see if it meets its retrieval criteria (via Query Protocol)
 - The client schedules a `Data Transfer Pull Request` passing the `RetrievalDealProposal` as a voucher.
-- The provider validates the proposal and rejects it if it is invalid
-- If the proposal is valid, the provider responds with an accept message and begins monitoring the data transfer process
-- The client creates a payment channel as necessary and a lane, ensures there are free funds in the channel
-- The provider unseals the sector as necessary
-- The provider monitors data transfer as it sends blocks over the protocol, until it requires payment
-- When the provider requires payment, it pauses the data transfer and sends a request for payment as an intermediate voucher
-- The client receives the request for payment
-- The client creates and stores payment voucher off-chain
-- The client responds to provider with a reference to the payment voucher, sent as an intermediate voucher
-- The provider redeems the payment voucher off-chain
-- The provider resumes both the request and sending data
-- The process continues until the end of the query
+- The provider validates the proposal and rejects it if it is invalid.
+- If the proposal is valid, the provider responds with an accept message and begins monitoring the data transfer process.
+- The client creates a payment channel as necessary and a "lane" and ensures there are enough funds in the channel.
+- The provider unseals the sector as necessary.
+- The provider monitors data transfer as it sends blocks over the protocol, until it requires payment.
+- When the provider requires payment, it pauses the data transfer and sends a request for payment as an intermediate voucher.
+- The client receives the request for payment.
+- The client creates and stores a payment voucher off-chain.
+- The client responds to the provider with a reference to the payment voucher, sent as an intermediate voucher (i.e., acknowledging receipt of a part of the data and channel or lane value).
+- The provider redeems the payment voucher off-chain.
```

Comment:
We should remove the `redeem` word here -- it's confusing.

I would change this step to:
`The provider validates the voucher sent by the client and saves it to be redeemed on-chain later"